src/components/Table/Table.scss

```diff
@@ -43,11 +42,11 @@ $basic-border: 1px solid #e2e2e2;
       color: #646464;
 
       .anticon-caret-up::before {
-        content: "\e607";
+        content: '\e607';
```

Comment:
In a followup I plan on piping the SCSS through prettier as well which is where these formatting changes come from.